src/thor/bidirectional_astar.cc

```diff
@@ -1276,6 +1276,9 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
     }
 
     // recost edges in final path; ignore access restrictions
+    // TODO: actually we should not ignore access restrictions: if the reverse path
+    //   traversed a closed edge due to time restrictions, we could do a mini traversal
+    //   to circumvent the closed edge(s)
```

Comment:
it's not the nicest patch, but still nicer than navigating over an edge we know is closed